scripts/fn_caves.sqf

```diff
@@ -0,0 +1,96 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Mike
+ * Handles night-time cycle in caves, removes Map & MicroDAGR if player has one.
+ * Will check an array of markers instead of running multiple per frame handlers on players.
+ * The time of day and custom ACRE signal can be edited in the defines.
+ *
+ * Call from initPlayerLocal.sqf.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ * 1: Markers <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_player, ["Marker_1"]] call FUNC(caves);
+ * [_player, ["Marker_1", "Marker_2"]] call FUNC(caves);
+ */
+
+params ["_player", "_markers"];
+
+#define DATE_TIME 23
+#define CUSTOM_SIGNAL [0.05, -70]
```

Comment:
Why not parameters? What if you want multiple caves with different levels of light and/or signal degradation.